.circleci/config.yml

```diff
@@ -326,6 +321,141 @@ jobs:
+
+            # Store elastic user password
+            echo 'ELASTIC_PASSWORD="$(kubectl get secret data-lake-es-elastic-user -o jsonpath="{.data.elastic}" | base64 -d)"' >> $BASH_ENV
+            source $BASH_ENV
+
+            # Execute an index creation request in the elasticsearch container
+            kubectl exec data-lake-es-default-0 --container elasticsearch -- \
+              curl -ks -X PUT "https://elastic:${ELASTIC_PASSWORD}@localhost:9200/statements?pretty"
+      - run:
+          name: Deploy Ralph
+          command: |
+            cd src/helm
+            helm dependency build ralph/
+            kubectl create secret generic ralph-auth-secret --from-file=auth.json=manifests/auth-demo.json
+            sed -i -e "s|<PASSWORD>|${ELASTIC_PASSWORD}|g" manifests/ralph-env-secret.yaml
+            kubectl apply -f manifests/ralph-env-secret.yaml
+            kubectl apply -f manifests/ralph-env-cm.yaml
+            helm install -f development.yaml lrs ralph/
+
+            # Test the deployment/switch
+            kubectl wait --for=condition=ready pod --selector=app=ralph --timeout=120s
+
+             # Test the LRS server health
+            kubectl exec svc/lrs-ralph -- curl -s -X GET "http://localhost:8080/__heartbeat__" | \
+              grep '"database":"ok"'
+
+  publish-helm:
+    docker:
+      - image: cimg/base:current
+        auth:
+          username: $DOCKER_HUB_USER
+          password: $DOCKER_HUB_PASSWORD
+    working_directory: ~/fun
+    steps:
+      # Checkout repository sources
+      - checkout
+      - *docker-login
+      - run:
+          name: Install the kubectl client, helm and k3d
+          command: |
+            export HELM_RELEASE="v3.13.2"
+            curl -Lo "/tmp/helm.tar.gz" "https://get.helm.sh/helm-${HELM_RELEASE}-linux-amd64.tar.gz"
+            curl -sL "/tmp/helm.tar.gz.sha256sum" "https://get.helm.sh/helm-${HELM_RELEASE}-linux-amd64.tar.gz.sha256sum" | \
+              sed "s|helm-${HELM_RELEASE}-linux-amd64.tar.gz|/tmp/helm.tar.gz|" | \
+              sha256sum --check
+            tar -xf /tmp/helm.tar.gz --strip-components=1 -C ${HOME}/bin linux-amd64/helm
+            chmod 755 "${HOME}/bin/helm"
+      - run:
+          name: Get Helm chart target version from git tag
+          command: |
+            echo "export HELM_TAG=$(echo ${CIRCLE_TAG} | sed 's|^helm-v||')" >> "$BASH_ENV"
+            # Display:
+            # - HELM_TAG: 1.1.2 (Git tag helm-v1.1.2)
+            echo "HELM_TAG: ${HELM_TAG} (Git ${CIRCLE_TAG})"
+      - run:
+          name: Package Ralph Helm chart
+          command: | 
+            cd src/helm
+            helm dependency build ralph/
+            helm package ralph
+            # Check that chart version matches the tag
+            ls | grep ralph-${HELM_TAG}.tgz
+      - run:
+          name: Upload built package to DockerHub
+          command: |
+            cd src/helm
```

Comment:
Instead of doing it twice, maybe you can update the working directory instead?

Reply:
Good point!